atom/index.js

```diff
@@ -11,69 +32,56 @@ export let atom = (initialValue, level) => {
       }
       return $atom.value
     },
-    l: level || 0,
+    isEqual: Object.is,
```

Comment:
By moving to `eq` or `is` we can save some size. This API is very rare, so it is OK to use some cryptic name.